navigation/api/src/main/resources/ApplicationResources.properties

```diff
@@ -0,0 +1,131 @@
+phenotips.livetable.filter.placeholder=Type to filter...
+patient.livetable.doc.name=Identifier
+patient.livetable.doc.creator=Reported by
+patient.livetable.doc.author=Last modified by
+patient.livetable.doc.creationDate=Report date
+patient.livetable.doc.date=Last modification date
+patient.livetable.last_name=Last name
+patient.livetable.first_name=First name
+patient.livetable.gender=Gender
+patient.livetable.date_of_birth=Date of birth
+patient.livetable.health_card=Health card
+patient.livetable.relative_of=Relative of
+patient.livetable._actions=
+patient.livetable.emptyvalue=-
+patient.livetable.p_id=Identifier
+patient.livetable.congenital=Congenital
+patient.livetable.sex=Sex
+patient.livetable.global_age_of_onset=Onset
+patient.livetable.phenotype=Phenotype
+patient.livetable.negative_phenotype=Negative phenotype
+patient.livetable.omim_id=Disorder
+patient.livetable.identifier=Identifier
+patient.livetable.external_id=Report name
+patient.livetable.family_id=Family Id
+patient.livetable.protocol_id=Protocol Id
+patient.livetable.global_mode_of_inheritance=Global mode of inheritance
+patient.livetable.unaffected=Clinically normal
+patient.livetable.exam_date=Exam Date
+patient.livetable.relative=Relative
+patient.livetable.consanguinity=Consanguinity
+patient.livetable.omim_id=OMIM disorder
+patient.livetable.gestation=Gestation at delivery (weeks)
+patient.livetable.proband=Proband
+patient.livetable.in_vitro=In vitro
+patient.livetable.maternal_ethnicity=Maternal ethnicity
+patient.livetable.paternal_ethnicity=Paternal ethnicity
+patient.livetable.family_of=Pedigree number
+patient.livetable.miscarriages=Parents with at least 3 miscarriages
+
+admin.phenotips=PhenoTips
```

Comment:
These shouldn't be duplicated inside this repo. Let's find a way to structure the maven projects so that this is not necessary.